src/sentry/snuba/metrics/query_builder.py

```diff
@@ -349,6 +353,10 @@ def _build_totals_and_series_queries(
             granularity=Granularity(rollup),
             orderby=orderby,
         )
+
+        if not self._query_definition.include_series:
```

Comment:
I needed query_definition here. Since I saw that we're already passing around quite a lot of state between static methods, I thought it would be appropriate to refactor this class such that it stores the query definition in `__init__` and does everything else lazily